charts/redpanda/templates/_configmap.tpl

```diff
@@ -0,0 +1,463 @@
+{{/*
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/}}
+
+{{- define "configmap-content-no-seed" -}}
+{{- /*
+  configmap content without seed list.
+*/ -}}
+{{- $root := . }}
+{{- $values := .Values }}
+
+{{- /*
+  It's impossible to do a rolling upgrade from not-tls-enabled rpc to tls-enabled rpc.
+*/ -}}
+{{- $check := list
```

Comment:
NIT

The `check` variable isn't meaningful. I don't even understand why we need at least 23.1.2 or 22.3.13 or 22.2.10? Could we name the feature or bugfix that is included in that version? 

```suggestion
{{- $check := list
```

Reply:
Will review, this is a great time to make these kind of changes.